ui/pages/settings/index.js

```diff
@@ -657,19 +661,314 @@ function Admins() {
   )
 }
 
-function Providers() {
+function AddAllowedDomainDialog({ setOpen, org }) {
```

Comment:
Instead of a dialog (too many modal dialogs can be exhausting!), we can use a simple "tag" (or "badge") input box here.

Reply:
Updated